Spanner/src/ValueMapper.php

```diff
@@ -324,6 +326,7 @@ private function decodeValue($value, array $type)
                 }
                 break;
 
+            case self::TYPE_FLOAT32:
```

Comment:
I think the error below should be corrected to include FLOAT32 too in the message

Reply:
Fixed it in c260a86